File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Python/PRAK404-2310817120009-AlysaArmelia.py
while True:
    print("Pilihan program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Keluar")

    pilihan = int(input("Masukkan pilihan Anda: "))

    if pilihan == 5:
        NAMAANDA = "Alysa Armelia"
        print("Terimakasih, telah menggunakan kalkulator", NAMAANDA)
        break

    elif pilihan > 5 or pilihan < 1:
        print("Pilihan Anda salah, silakan coba lagi. \n")
        continue

    bilangan1 = float(input("Masukkan nilai pertama: "))
    bilangan2 = float(input("Masukkan bilangan kedua: "))

    if pilihan == 1:
        hasil = bilangan1 + bilangan2
        print(f"Hasil penjumlahan antara {bilangan1:.2f} dan {bilangan2:.2f} adalah {hasil:.2f}")

    elif pilihan == 2:
        hasil = bilangan1 - bilangan2
        print(f"Hasil pengurangan antara {bilangan1:.2f} dan {bilangan2:.2f} adalah {hasil:.2f}")

    elif pilihan == 3:
        hasil = bilangan1 * bilangan2
        print(f"Hasil perkalian antara {bilangan1:.2f} dan {bilangan2:.2f} adalah {hasil:.2f} \n")

    elif pilihan == 4:

        if bilangan2 != 0:
            hasil = bilangan1 / bilangan2
            print(f"Hasil pembagian antara {bilangan1:.2f} dan {bilangan2:.2f} adalah {hasil:.2f}")

        else:
            print("Pembagian dengan nol tidak diizinkan.")

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Bahasa C/PRAK404-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main() {
    int pilihan_program;
    char NAMAANDA[20] = "Alysa Armelia";
    float nilai1;
    float nilai2;
    float keseluruhan;

    while (1) 
    {
        printf("Pilihan program \n");
        printf("1. Penjumlahan \n");
        printf("2. Pengurangan \n");
        printf("3. Perkalian \n");
        printf("4. Pembagian \n");
        printf("5. exit \n");

        printf("Masukkan pilihan Anda: ");
        scanf("%d", & pilihan_program);
    
        if (pilihan_program == 5) {
            printf("Terimakasih, telah menggunakan kalkulator %s", NAMAANDA);
            break;

        }else if(pilihan_program > 5 || pilihan_program < 1) {
            printf("Pilihan Anda salah, silakan coba lagi.\n");
            printf("\n");
            continue;
        }
        printf("Masukkan nilai pertama: ");
        scanf("%f", &nilai1);
        printf("Masukkan bilangan kedua: ");
        scanf("%f", &nilai2);

        if (pilihan_program == 1) {
            keseluruhan = nilai1 + nilai2;
            printf("Hasil penjumlahan antara %.2f dan %.2f adalah %.2f\n", nilai1, nilai2, keseluruhan);

        } else if (pilihan_program == 2) {
            keseluruhan = nilai1 - nilai2;
            printf("Hasil pengurangan antara %.2f dan %.2f adalah %.2f\n", nilai1, nilai2, keseluruhan);

        } else if (pilihan_program == 3) {
            keseluruhan = nilai1 * nilai2;
            printf("Hasil perkalian antara %.2f dan %.2f adalah %.2f\n", nilai1, nilai2, keseluruhan);
            printf("\n");

        } else if (pilihan_program == 4) {

            if (nilai2 != 0) {
                keseluruhan = nilai1 / nilai2;
                printf("Hasil pembagian antara %.2f dan %.2f adalah %.2f\n", nilai1, nilai2, keseluruhan);

            } else {
                printf("Pembagian dengan nol tidak diizinkan.\n");
            }
        } 
    }
return 0;
}